addons/spreadsheet/static/src/chart/odoo_chart/odoo_chart.js

```diff
@@ -102,16 +101,18 @@ export class OdooChart extends AbstractChart {
     }
 
     /**
+     * @param {string} sheetId
      * @returns {OdooChart}
      */
-    copyForSheetId() {
+    copyForSheetId(sheetId) {
         return this;
     }
 
     /**
+     * @param {string} sheetId
      * @returns {OdooChart}
      */
-    copyInSheetId() {
+    copyInSheetId(sheetId) {
```

Comment:
Useless

Reply:
remainder of the first approach. Taht being said, I think it's quite bad to yield the same object when calling a "copy" function. As far as I can see, we only create those items to extract their definitions later on - I think we should change thoses functions so the yield a definition directly but the copy/paste behaviour makes it not as easy as I thought